tensorflow/inference/docker/build_artifacts/sagemaker/serve.py

```diff
@@ -137,22 +152,34 @@ def _setup_gunicorn(self):
                         raise ChildProcessError('failed to install required packages.')
 
         gunicorn_command = (
-            'gunicorn -b unix:/tmp/gunicorn.sock -k gevent --chdir /sagemaker '
-            '{}{} -e TFS_GRPC_PORT={} -e SAGEMAKER_MULTI_MODEL={} -e SAGEMAKER_SAFE_PORT_RANGE={} '
-            'python_service:app').format(python_path_option, ','.join(python_path_content),
-                                         self._tfs_grpc_port, self._tfs_enable_multi_model_endpoint,
+            "gunicorn -b unix:/tmp/gunicorn.sock -k {} --chdir /sagemaker "
+            "{}{} -e TFS_GRPC_PORT_RANGE={} -e TFS_REST_PORT_RANGE={} "
```

Comment:
can we use SAGEMAKER_GUNICORN_WORKERS and SAGEMAKER_GUNICORN_THREADS to configure gunicorn here as well?

Reply:
yep, added back